mitodl/mitxpro
# Upgraded wagtail to 2.9.3, added image rendition caching

#### Pre-Flight checklist


#### What are the relevant tickets?
Partially addresses https://github.com/mitodl/mitxpro/issues/2019

#### What's this PR do?
Upgrades Wagtail to 2.9.3, which includes image rendition caching, and adds configuration to perform that caching

#### How should this be manually tested?
Since this involves a bunch of library upgrades, a good site-wide smoke test won't hurt.

To test the caching specifically:
- Start a Django shell
- Run this code snippet to check the contents of Redis:
    ```python
    from django.conf import settings
    import redis
    client = redis.from_url(settings.WAGTAIL_CACHE_URL)
    client.execute_command("KEYS *")
    ```
- Load a page in the web app which has CMS images (e.g.: the catalog or a product page)
- Run the snippet above once again. There should be new keys prefixed with "wag"

#### Any background context you want to provide?
We are making an effort to improve caching of these images in Fastly. In an ideal world, these images wouldn't end up in the Redis cache at all. This can be considered a fallback in case the caching in Fastly falls through. If Wagtail renders an image, it will automatically be saved in the Redis cache. Subsequent requests for the image will be served from there, saving a significant amount of time/memory
